### PR TITLE
Fix broken builds

### PR DESCRIPTION
**Pull request by devnotfound**
https://github.com/sohrab-/mule4-openshift/issues/2

### CODE REVIEW

File: image/classpath [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
+mule-app-secrets
 mule/conf
-mule/lib/boot/*
-mule/lib/launcher/*
-mule/lib/opt/*
-mule/lib/mule/*
-mule/lib/patches/*
-mule/lib/user/*
\ No newline at end of file
+mule/lib/boot/commons-cli-1.2.jar
```
Comment (sohrab-) [inline image/classpath:4]: Why can't this be just `mule/lib/boot/*` so you don't have to update all those versions every time you update the mule runtime version?

File: image/classpath [hunk 2]
```diff
@@ -1,9 +1,24 @@
 mule-licence
 mule-app-properties
+mule-app-secrets
```
Comment (sohrab-) [inline image/classpath]: This is a bit too opinionated for this repository.

**Review by sohrab- — CHANGES_REQUESTED** [review]

File: image/classpath (hunk 1 above)
Comment (devnotfound) [reply image/classpath:4]: Yea, I tried that, the jars don't get added in the classpath. Perhaps its a openjdk convention like documented here -> https://github.com/fabric8io-images/java/tree/master/images/jboss/openjdk8/jdk

> JAVA_LIB_DIR directory holding the Java jar files as well an optional classpath file which holds the classpath. Either as a single line classpath (colon separated) or with jar files listed line-by-line. If not set JAVA_LIB_DIR is the same as JAVA_APP_DIR.

File: image/classpath (hunk 2 above)
Comment (devnotfound) [reply image/classpath]: Not sure why, but sorted.

**Pull request merged**